homie-controller/src/lib.rs

```diff
@@ -124,6 +124,12 @@ impl HomieEventLoop {
     }
 }
 
+struct PublishResponse {
```

Comment:
At least derive `Debug`, might as well also derive `Eq`, `PartialEq`, `Clone`.

Reply:
I doubt that we will ever need to debug this, since it's an internal struct that lives for about 2 lines before being consumed.

I will add a docstring saying what it's used for, and we can add derives when we need them for tests or make the function public.

Reply:
It might come in handy for tests in the future, but I guess I can always add it then.